src/Features/Core/Portable/FindUsages/DefinitionItemFactory.cs

```diff
@@ -151,75 +116,108 @@ internal static class DefinitionItemExtensions
 
             var properties = GetProperties(definition, isPrimary);
 
-            if (sourceLocations.IsDefault || definition.IsTupleType())
```

Comment:
did we lose this IsTupleType check?  i know it was added recently to make sure goto-def on a tuple type (like `ValueTuple<A,B>`) worked.

Reply:
I'll test tuple.

Reply:
GTD works without this condition.